### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by jugosack**
- [ ] Create pages and component folder
- [ ] Use react-router-dom to navigate between Books.jsx & Categories.jsx pages
- [ ] Create Book.jsx, Form.jsx, Navbar.jsx component
- [ ] Use Prop-types for validation
- [ ] Linter check

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Categories from './pages/Categories';
+import NotFound from './pages/NotFound';
+
+function App() {
+  return (
+    <>
+      <Navbar />
+      <div className="container_1">
+        <Routes>
+          <Route path="/" element={<Books />} />
```
Comment (codepantha) [inline src/App.js]: - [ ] According to the requirements, the root `/` route should show a `<Home />` component which in turn will have the `<Books>` component and the `<Navbar />` component

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/prop-types */
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (codepantha) [inline src/components/Navbar.jsx]: - [ ] Please do not disable linters. They exist to help you adhere to strict guidelines 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @jugosack 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Set up application with components 📌 
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please remove boilerplate code such as `reportWebVitals.js` and `setUpTests.js` because they are not necessary for your project 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @jugosack , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## highLights 
:+1: no linter errors 
:+1: files structure

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**